kformat/kproperty.py

```diff
@@ -1,15 +1,22 @@
 import abc
+import enum
 from datetime import date, time
 from typing import Optional, Set
 
-from .exception import InvalidLengthError
+from .exception import InvalidLengthError, UnicodeKFormatError
 
 __all__ = ['AN', 'N']
 
 
 TYPES = Set[type]
 
 
+class Errors(enum.Enum):
```

Comment:
```suggestion
class UnicodeErrorHandler(enum.Enum):
```

이게 Unicode Error가 발생했을 때 처리하는 방식에 대한 분류이기 때문에 이런 이름이 좀 더 좋을 것 같다고 생각했습니다. 어떠세요?

Reply:
이것도 조금 애매했었는데, 좋은 이름인 것 같습니다. 변경했습니다!